src/atoms/Tooltip/index.js

```diff
@@ -69,27 +74,31 @@ class Tooltip extends React.Component {
       tooltipIconSize,
     } = this.props;
 
+    const { visible, modalIsOpen } = this.state;
+
     return (
       <span>
         <span
           onClick={revealOnClick ? this.handleClick : this.openModal}
           className={classnames(
             !revealOnClick && styles['tooltip-wrapper'],
-            this.state.visible && styles.reveal,
+            visible && styles.reveal,
             inline && styles[`inline-${inline}`],
             className
           )}
           ref={this.setWrapperRef}
         >
           {
-            text ||
-              <Icon
-                icon='tooltip'
-                className={styles['tooltip']}
-                inline={inline}
-                height={`${tooltipIconSize}px`}
-                width={`${tooltipIconSize}px`}
-              />
+            text
+              || (
```

Comment:
yeah all of these `||` or `&&` updates are a bit annoying

Reply:
Agreed. I've changed them all back to the way they were.